xsnippet_api/middlewares/__init__.py

```diff
@@ -8,7 +8,7 @@
     :license: MIT, see LICENSE for details
 """
 
-from .auth import auth
+from . import auth
```

Comment:
Hm.. I think there's no value in preserving this importing stuff in case these stuff is modules. What do you say about making `__init__.py` empty?